src/main/java/it/gov/pagopa/pu/workflow/wf/pagopa/send/wfsendnotification/SendNotificationDateRetrieveWF.java

```diff
@@ -0,0 +1,31 @@
+package it.gov.pagopa.pu.workflow.wf.pagopa.send.wfsendnotification;
```

Comment:
let's introduce a separate package in order to have 1 package per wf
```suggestion
package it.gov.pagopa.pu.workflow.wf.pagopa.send.wfretrievedt;
```